packages/jaeger-ui/src/model/find-trace-name.test.js

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { getTraceName } from './trace-viewer';
+
+describe('getTraceName', () => {
+  const firstSpanId = 'firstSpanId';
+  const secondSpanId = 'secondSpanId';
+  const thirdSpanId = 'thirdSpanId';
+  const remoteSpanId = 'remoteSpanId';
+
+  const serviceName = 'serviceName';
+  const operationName = 'operationName';
+
+  const spansWithNoRoots = [
+    {
+      spanID: firstSpanId,
+      startTime: 1583758690000,
```

Comment:
I recommend defining a constant `t` and using deltas like `t+10` to vary the timestamps.

Reply:
Thanks, [fixed](https://github.com/jaegertracing/jaeger-ui/pull/541/commits/579300519e0a8d2f73ab2f09bcf1187439b38da1).